src/app/shared/stack-blitz/stack-blitz-writer.ts

```diff
@@ -190,27 +190,28 @@ export class StackBlitzWriter {
       fileContent = fileContent.replace(/material-docs-example/g, data.selectorName);
       fileContent = fileContent.replace(/{{version}}/g, VERSION.full);
     } else if (fileName === 'src/main.ts') {
+      const joinedComponentNames = data.componentNames.join(', ');
```

Comment:
I don't see a matching change that renames `ExampleData.componentName` to `ExampleData.componentNames` in https://github.com/angular/components/pull/18547?

Oh, I tracked it down to PR https://github.com/angular/components/pull/18498.